includes/woocommerce-template.php

```diff
@@ -151,6 +159,19 @@ function dw_override_checkout_fields($fields) {
 		'required' => true,
 	];
 
+	$newOrderFields['order'][OrderManager::KEY_DELAY_REASON] = [
+		'type'     => 'select',
+		'options'  => [
+			''        => __('Kies een reden van uitstel'),
+			'medical' => __(DELAY_REASONS['medical']),
+			'service' => __(DELAY_REASONS['service']),
+			'study'   => __(DELAY_REASONS['study']),
+			'prison'  => __(DELAY_REASONS['prison'])
```

Comment:
This can be made more extendable in case of extra delay reasons with a forach on self::DELAY_REASONS.